nautobot/docs/user-guides/getting-started-nautobot-ui.md

```diff
@@ -0,0 +1,606 @@
+# Getting Nautobot Up and Running in the Web UI
+
+## Introduction and Scope
+
+The audience for this user guide is a user new to Nautobot. 
+It is intended to assist the user with:
+* Understanding the prerequisites to adding a new Device
```

Comment:
Please add a line break before all lists. It is preventing this list from rendering correctly.